PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py

```diff
@@ -108,6 +108,7 @@
         btagPNetB = Var("?bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralDiscriminatorsJetTags:BvsAll')>0?bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralDiscriminatorsJetTags:BvsAll'):-1",float,precision=10,doc="ParticleNet b vs. udscg"),
+        btagPNetCvNotB = Var("?(bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probuds')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probg')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probmu')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probele')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h2p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup3h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup3h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum1h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum1h2p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum3h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum3h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probq')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probg'))>0?(bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probc'))/(bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probuds')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probg')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probmu')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probele')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup1h2p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup3h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaup3h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum1h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum1h2p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum3h0p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probtaum3h1p')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probq')+bDiscriminator('pfParticleNetFromMiniAODAK4PuppiCentralJetTags:probg')):-1",float,precision=10,doc="ParticleNet C vs notB"),
```

Comment:
it looks like an addition to mainstream NANO ? if it is, it should go in customisation instead

Reply:
this is indeed we want to add to mainstream nano due to the SF correction/correct c tagger for analyzer

@SWuchterl,@mondalspandan  please confirm

Reply:
@vlimant We want this in the central Nano, indeed. CvsL seems to be inadequate for charm tagging in case of PNet, since separate lepton classes exist in PNet and true leptons often have a reasonably high CvsL score. Therefore, for future Nanos we want to support the CvNotB node instead of CvL. Having both nodes for the moment would help us fall back in case additional issues arise.

Reply:
at the very least `btagPNetCvNotB` should be added to nanoDQM_cfi.py after `btagPNetB`